ocs_ci/helpers/helpers.py

```
@@ -6599,48 +6599,70 @@ def remove_toleration():
+            try:
+                cron_obj = OCP(
+                    kind=constants.RECLAIMSPACECRONJOB,
+                    namespace=pvc_obj.namespace,
+                    resource_name=cron_job_name,
+                )
+                cron_obj.get()
+                logger.info(
+                    f"Found ReclaimSpaceCronJob '{cron_job_name}' for PVC '{pvc_obj.name}'"
+                )
+                return cron_obj
+            except Exception as ex:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)